requirements.txt

```diff
@@ -4,25 +4,26 @@
 trio==0.19.0
 
 # Docs
-mkdocs==1.2.3
+mkdocs==1.3.0
 mkdocs-autorefs==0.3.1
 mkdocs-material==8.2.3
 mkdocs-material-extensions==1.0.3
 mkdocstrings==0.17.0
+jinja2==3.0.3  # See: https://github.com/mkdocs/mkdocs/issues/2799
 
 # Packaging
-twine==3.8.0
-wheel==0.37.1
+twine
+wheel
 
 # Tests & Linting
 anyio==3.5.0
 autoflake==1.4
-black==22.1.0
+black==22.3.0
 coverage==6.2
-flake8==4.0.1
+flake8==3.9.2
 isort==5.10.1
 mypy==0.931
 pytest==7.0.1
-pytest-httpbin==1.0.1
+pytest-httpbin==1.0.2
```

Comment:
```suggestion
pytest-httpbin==2.0.0rc1
```

Reply:
Thanks! But the issue seems to be with httpbin itself. It imports `BaseResponse` but werkzeug==2.1.* released on March 28th removed it in favor of `Response`.